elibrary-micro/src/main/resources/application-prod.properties

```diff
@@ -0,0 +1,27 @@
+# Database config
+spring.datasource.url                     = jdbc:postgresql://jFeeSoft/jfs-operations
+spring.datasource.driver-class-name       = org.postgresql.Driver
+spring.datasource.username                = jfs-operations
```

Comment:
Please change `jfs-operations` to `elibrary-micro`.

Reply:
done